rules/falco_rules.yaml

```diff
@@ -1029,6 +1029,13 @@
                    fd.name startswith "/etc/rancher-dns")
              )
 
+- macro: jboss_in_container_writing_passwd
+  condition: >
+    ((proc.cmdline="run-java.sh /opt/jboss/container/java/run/run-java.sh"
```

Comment:
the `proc.cmdline` looks strange, the `args[1]` is the abs path of the `exe`?

Reply:
It does look strange, but the policy event did have that as the full cmdline. Here's an example:

```
File below /etc opened for writing (user=<NA> command=run-java.sh /opt/jboss/container/java/run/run-java.sh parent=java pcmdline=java -javaagent:/opt/jboss/container/jolokia/jolokia.jar=config=/opt/jboss/container/jolokia/etc/jolokia.properties....
```

And for reference, the output field of that rule is:

```
output: "File below /etc opened for writing (user=%user.name command=%proc.cmdline parent=%proc.pname pcmdline=%proc.pcmdline file=%fd.name program=%proc.name gparent=%proc.aname[2] ggparent=%proc.aname[3] gggparent=%proc.aname[4])"
```